Source/Login/Backend/identities/tenants/configuration.go

```diff
@@ -0,0 +1,5 @@
+package tenants
+
+type Configuration interface {
+	TenantNamesMap() map[TenantID]string
```

Comment:
Change name

```suggestion
	TenantNames() map[TenantID]string
```

Reply:
Ohh, missed this suggestion so I did it myself in [this commit](https://github.com/dolittle-platform/Authentication/pull/8/commits/7a1fde4587086b88dc5bd09e77973e1a4bb01819)